akka-distributed-data/src/main/scala/akka/cluster/ddata/Replicator.scala

```diff
@@ -2302,6 +2302,7 @@ final class Replicator(settings: ReplicatorSettings) extends Actor with ActorLog
       val orderedNodes =
         if (requiresCausalDeliveryOfDeltas) reachableNodes.toVector.sorted ++ unreachable.toVector.sorted
         else scala.util.Random.shuffle(reachableNodes.toVector) ++ scala.util.Random.shuffle(unreachable.toVector)
+      // FIXME else reachableNodes.toVector.sorted ++ unreachable.toVector.sorted
```

Comment:
here is an experiment to simulate writing to different nodes than reading (reading is using reverse)